### PR TITLE
UIImage with color

### PR DESCRIPTION
**Pull request by Jayxiang**
UIImage with color uses .alwaysOriginal rendering mode.
## Checklist

- [x] I checked the [**Contributing Guidelines**](https://github.com/SwifterSwift/SwifterSwift/blob/master/CONTRIBUTING.md) before creating this request.
- [ ] New extensions are written in Swift 5.0.
- [ ] New extensions support iOS 10.0+ / tvOS 9.0+ / macOS 10.10+ / watchOS 2.0+, or use `@available` if not.
- [ ] I have added tests for new extensions, and they passed.
- [ ] All extensions have a **clear** comments explaining their functionality, all parameters and return type in English.
- [ ] All extensions are declared as **public**.
- [x] I have added a [changelog](https://github.com/SwifterSwift/SwifterSwift/blob/master/CHANGELOG_GUIDELINES.md) entry describing my changes.

### CODE REVIEW

File: Sources/SwifterSwift/UIKit/UIImageExtensions.swift [hunk 1]
```diff
@@ -319,6 +319,16 @@ public extension UIImage {
...
         return jpegData(compressionQuality: compressionQuality)?.base64EncodedString()
     }
+
+    /// SwifterSwift: UIImage with color uses .alwaysOriginal rendering mode.
+    ///
+    /// - Parameters:
+    ///   - color: Color of image.
+    /// - Returns: UIImage with color.
+    @available(iOS 13.0, tvOS 13.0, watchOS 6.0, *)
+    func alwaysOriginal(withColor: UIColor) -> UIImage {
```
Comment (guykogus) [inline Sources/SwifterSwift/UIKit/UIImageExtensions.swift:329]: Keep the name aligned with the internal naming.
```suggestion
    func withAlwaysOriginalTintColor(_ color: UIColor) -> UIImage {
```

File: Tests/UIKitTests/UIImageExtensionsTests.swift [hunk 2]
```diff
@@ -242,6 +242,12 @@ final class UIImageExtensionsTests: XCTestCase {
...
     }
+
+    @available(iOS 13.0, tvOS 13.0, watchOS 6.0, *)
+    func testAlwaysOriginalWithColor() {
+        let image = UIImage(color: .blue, size: CGSize(width: 20, height: 20))
+        XCTAssertEqual(image.alwaysOriginal(withColor: .red), image.withTintColor(.red, renderingMode: .alwaysOriginal))
```
Comment (guykogus) [inline Tests/UIKitTests/UIImageExtensionsTests.swift:249]: You're only testing the internal implementation. If you didn't know what it was, what could you test here? Can you try and check the colour of the image, for example?

**Review by guykogus — CHANGES_REQUESTED** [review]
You're missing a CHANGELOG entry and the tests need some work.

P.S. Sorry for the very late review 😬

File: Tests/UIKitTests/UIImageExtensionsTests.swift (hunk 2 above)
Comment (Jayxiang) [reply Tests/UIKitTests/UIImageExtensionsTests.swift]: What should I do, I'm not sure.

**Review by guykogus — APPROVED** [review]

**Review by guykogus — APPROVED** [review]

**Pull request merged**